### ISSUE TITLE
Would you please support the binary secret key?

### ISSUE THREAD

**Issue by jemygraw** [Laerte/aes_pkcs5#13]
Currently the secret key must be a utf-8 encoded string but in some cases we have to use the binary secret key.
Thx if possible. For the current workaround solution, I implements my own class for anybody who may need it.

```python
from cryptography.hazmat.backends import default_backend
from cryptography.hazmat.primitives.ciphers.algorithms import AES
from cryptography.hazmat.primitives.ciphers.modes import ECB

from aes_pkcs5.algorithms import AESCommon, Cipher, OUTPUT_FORMATS


class AESECBPKCS5Padding(AESCommon):
    """
    Implements AES algorithm with ECB mode of operation and padding scheme PKCS5.
    """

    def __init__(self, key: bytes, output_format: str):
        super().__init__('0', output_format)
        self._key = key

        if output_format not in OUTPUT_FORMATS:
            raise NotImplementedError(
                f"Support for output format: {output_format} is not implemented"
            )

        self._output_format = output_format

    def _get_cipher(self):
        """Return AES/CBC/PKCS5Padding Cipher"""
        return Cipher(AES(self._key), mode=ECB(), backend=default_backend())

```

**Comment by Laerte** [comment]
Hello @jemygraw I will take a look and implement this feature in the next version, thanks!

**Comment by Laerte** [comment]
Implemented on 1.0.3 version.
